main.go

```diff
@@ -67,13 +70,28 @@ func main() {
 
 	// connect to exec api end point(websocket with json-rpc)
 	r.GET("/connect", func(c *gin.Context) {
+		var userToken string
+		if client.UseUserToken {
```

Comment:
Previously, che-machine-exec was built in implementation independence way. Now you introduce direct dependency to k8s implementation.
It's fully OK since we don't have docker infra anymore, but maybe it makes sense to extract authentication things independent from k8s...

As I already commented, UseUserToken could be read from `config`...

`client.UserTokenAttr` could be moved somewhere in constant... If you take a look on its usages, k8s client does not use it at all:
![Screenshot_20200317_144313](https://user-images.githubusercontent.com/5887312/76857232-b0761f80-685d-11ea-837c-f232f4c59c02.png)